client/API.md

```diff
@@ -2116,6 +2117,62 @@ POST /kapacitor/v1/config/influxdb/remote
 | 403  | Config override service not enabled                       |
 | 404  | The specified configuration section/option does not exist |
 
+## Storage
+
+Kapacitor exposes some operations that can be performed on the underlying storage.
+
+>WARNING: Everything storage operation is directly manipulating the underlying storage database.
+Always make a backup of the database before performing any of these operations.
+
+### Backing up the Storage
+
+Making a GET request to `/kapacitor/v1/storage/backup` will return a dump of the Kapacitor database.
+To restore from a backup replace the `kapacitor.db` file with the contents of the backup request.
+
+```
+# Create a backup.
+curl http://localhost:9092/kapacitor/v1/storage/backup > kapacitor.db
```

Comment:
Agreed, I'll put this together. I don't want a subcommand for the rebuild action though, I figure those are best only in the API so that you have to read and be aware of how they work.